extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -17,6 +17,10 @@
 @import './amp-story-shadow-reset.css';
 @import './amp-story-share.css';
 
+:host {
+  visibility: inherit !important;
```

Comment:
It's worth adding an inline comment to say this propagates the visibility value from its shadow host for a11y